CHANGELOG.md

```diff
@@ -37,9 +37,9 @@
 - nursery/enumerate-pe-sections-in-dotnet @mr-tz
 - nursery/destroy-software-breakpoint-capability echernofsky@google.com
 - nursery/send-data-to-internet michael.hunhoff@mandiant.com
--
```

Comment:
i think we need this to be here for our bot to update the rule list, would you please revert this line deletion?

(thanks for trying to clean this up)